src/main/java/org/mule/extensions/java/internal/metadata/ExecutableElementTypeResolver.java

```diff
@@ -103,6 +101,9 @@ public MetadataType getOutputType(MetadataContext context, ExecutableIdentifier
         ? ((Method) element).getGenericReturnType()
         : loadClass(key.getClazz());
 
+    if (output.getTypeName().equals(Object.class.getTypeName())) {
```

Comment:
what about the input metadata? We have to be consistent, If a parameter is Object then it has to be AnyType too.

Reply:
Good catch, I will apply the same on input parameters.